lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -22,7 +22,9 @@ def call(package, apply_password_parameter_check: false)
             return [ValidationError.new(:missing_manifest)]
           end
 
-          collate_manifest_errors(package, apply_password_parameter_check)
+          package.warnings << password_parameter_warning unless error_on_password_parameter
```

Comment:
Hey @mtsolakiszen ,

Wouldn't this add a warning for all app validations instead of only the ones that have a password param?